scripts/mods/mod_reforged/dynamic_spawns/parties/bandit_parties.nut

```diff
@@ -1,3 +1,6 @@
+local UpgradeChance = 0.5;
```

Comment:
Redundant as it is never used. If this was meant as a note, write it as a comment